configs/detection_light_model.yaml

```diff
@@ -31,15 +32,21 @@ trainer:
   optimizer:
     name: SGD
     params:
-      lr: 0.02
-      momentum: 0.937
-      weight_decay: 0.0005
-      dampening: 0.0
-      nesterov: true
+      lr: 0.0032           
```

Comment:
Just out of curiosity where do these values come from?

Reply:
Official YoloV6 finetune config